src/com/owncloud/android/ui/adapter/ExpandableUploadListAdapter.java

```diff
@@ -23,6 +23,7 @@
 import android.content.Context;
 import android.database.DataSetObserver;
 import android.graphics.Bitmap;
+import android.support.design.widget.Snackbar;
```

Comment:
May you haven't deleted the Toast dependence import, check the same in other classes too...